easybuild/easyconfigs/v/Virtuoso-opensource/Virtuoso-opensource-7.2.6.1-foss-2021a.eb

```diff
@@ -0,0 +1,71 @@
+##
+# This is a contribution from SIB Swiss Institute of Bioinformatics
+# Homepage:     https://www.sib.swiss/research-infrastructure/competence-centers/vital-it
+#
+# Authors::     Sebastien Moretti <sebastien.moretti@sib.swiss>
+#
+##
+easyblock = 'ConfigureMake'
+
+name = 'Virtuoso-opensource'
+version = '7.2.6.1'
+
+homepage = 'https://github.com/openlink/virtuoso-opensource'
+description = """Virtuoso is a high-performance and scalable Multi-Model RDBMS, Data
+Integration Middleware, Linked Data Deployment, and HTTP Application Server Platform."""
+software_license = 'LicenseGPLv2'
+
+toolchain = {'name': 'foss', 'version': '2021a'}
+
+sources = ['v%(version)s.tar.gz']
+source_urls = ['https://github.com/openlink/virtuoso-opensource/archive/refs/tags/']
+checksums = ['73dc3d0073d3d5694da075063b75a404649c027a1f5c635a25ede52713ff7942']
+
+builddependencies = [
+    ('Autotools', '20210128'),
+    ('flex',      '2.6.4'),
+    ('Bison',     '3.7.6'),
+    ('gperf',     '3.1'),
+]
+
+osdependencies = ['gawk']
```

Comment:
Why not use a proper dependency for `gawk` provided via EasyBuild?

Reply:
Done, I use gawk from EasyBuild see #14274